nc-news/src/Components/Articles.jsx

```diff
@@ -0,0 +1,26 @@
+import { useState, useEffect } from "react";
+import axios from "axios";
+import { ArticleCard } from "./ArticleCard";
+
+export const Articles = () => {
+    const [articles, setArticles] = useState([]);
+
+    useEffect(() => {
+        axios.get("https://robbies-articles-website.onrender.com/api/articles")
+        .then((response) => {
+            setArticles(response.data.articles)
+        })
+        .catch((err) => {
+            console.log(err);
```

Comment:
We should be doing something with this error to feedback to users in the case of an error occurring. How can we conditionally respond with an error component if we receive an error?